### PR TITLE
Mark libuv packages as shipping

### LINKED ISSUE #70: Libuv packages are marked as non-shipping
* Microsoft.AspNetCore.Internal.libuv-Darwin and Microsoft.AspNetCore.Internal.libuv-Windows are marked as non-shipping while the Linux one is marked as shipping. Consequently these packages don't get published to cidev \ vnext by Coherence and Coherence-Signed.

### PR DESCRIPTION
**Pull request by pranavkm**
Fixes #70

### CODE REVIEW

File: packages/packages.csv [hunk 1]
```diff
@@ -58,9 +58,9 @@ Microsoft.AspNetCore.HttpOverrides,ship
 Microsoft.AspNetCore.Identity,ship
 Microsoft.AspNetCore.Identity.EntityFrameworkCore,ship
 Microsoft.AspNetCore.IISPlatformHandler,noship
-Microsoft.AspNetCore.Internal.libuv-Darwin,noship
+Microsoft.AspNetCore.Internal.libuv-Darwin,ship
```
Comment (Eilon) [inline packages/packages.csv]: Hmm, why are these “Internal” ones “shipping”? This doesn’t look right.

File: packages/packages.csv [hunk 2]
```diff
@@ -58,9 +59,6 @@ Microsoft.AspNetCore.HttpOverrides,ship
 Microsoft.AspNetCore.Identity,ship
 Microsoft.AspNetCore.Identity.EntityFrameworkCore,ship
 Microsoft.AspNetCore.IISPlatformHandler,noship
-Microsoft.AspNetCore.Internal.libuv-Darwin,noship
-Microsoft.AspNetCore.Internal.libuv-Linux,ship
-Microsoft.AspNetCore.Internal.libuv-Windows,noship
```
Comment (BrennanConroy) [inline packages/packages.csv]: Shouldn't these be left as noship?

Comment (pranavkm) [inline packages/packages.csv]: We're going to completely skip the `libuv-build-windows` repo. If we want to preserve the internal packages in our build results, we should `libuv-package` to drop these packages in the artifacts directory.

Comment (Eilon) [inline packages/packages.csv]: Talked to @pranavkm about this and it's actually fine. These internal libuv packages are built and published via a different process, but this list here is only for things that are built as part of ASP.NET/EF, so it's not needed here.

**Comment by Eilon** [comment]
:shipit:

File: packages/packages.csv [hunk 3]
```diff
@@ -15,6 +15,7 @@ dotnet-sql-cache,ship
 dotnet-test-xunit,ext
 dotnet-user-secrets,ship
 dotnet-watch,ship
+Libuv,ship-ext
```
Comment (pranavkm) [inline packages/packages.csv]: Updated. Publishing Libuv and Sqlite to `ship-ext`. I'll speak to @joeloff and @phenning about picking up from this directory in addition to `ship` and `ext` for templates and tooling.

File: packages/packages.csv [hunk 4]
```diff
@@ -106,14 +104,14 @@ Microsoft.AspNetCore.SignalR.SqlServer,noship
 Microsoft.AspNetCore.StaticFiles,ship
 Microsoft.AspNetCore.TestHost,ship
 Microsoft.AspNetCore.Testing,noship
-Microsoft.AspNetCore.WebSockets.Client,ship
+Microsoft.AspNetCore.WebSockets.Client,noship
```
Comment (pranavkm) [inline packages/packages.csv]: cc @muratg  - this was part of https://github.com/aspnet/Coherence/commit/dd84f82db56ad63c3ff0b5e0122fe1ca250bcaaf that was unintentionally only checked in to dev.

**Comment by pranavkm** [comment]
Updated

File: packages/packages.csv (hunk 3 above)
Comment (joeloff) [inline packages/packages.csv]: I'll open a setup bug to add these to the feed installer, thanks

File: packages/packages.csv (hunk 2 above)
Comment (halter73) [inline packages/packages.csv]: As long as we still have "libuv-build-windows" in the `package-sources` artifacts it should still be easy to find the `Microsoft.AspNetCore.Internal.libuv-*` packages used to build the `Libuv` package.

**Comment by halter73** [comment]
:shipit:

File: packages/packages.csv (hunk 2 above)
Comment (pranavkm) [inline packages/packages.csv]: Spoke to @halter73 offline. I'll leave these as is and mark them as `noship`. He'll make a change to `libuv-package` to copy the nupkgs to the artifacts directory at which point we'll redo - https://github.com/aspnet/Coherence/pull/71/files#diff-941327d979d4d3fb4ad5068c28050da3L21

**Comment by pranavkm** [comment]
Updated

File: packages/packages.csv [hunk 5]
```diff
@@ -106,14 +107,14 @@ Microsoft.AspNetCore.SignalR.SqlServer,noship
...
 Microsoft.AspNetCore.WebSockets.Protocol,ship
 Microsoft.AspNetCore.WebSockets.Server,ship
 Microsoft.AspNetCore.WebUtilities,ship
 Microsoft.Cci,ext
 Microsoft.CodeAnalysis.*,ext
 Microsoft.CSharp,ext
-Microsoft.Data.Sqlite,ship
+Microsoft.Data.Sqlite,ship-ext
```
Comment (natemcmaster) [inline packages/packages.csv]: Why ship-ext? How is this different from "ship"?

Comment (Eilon) [inline packages/packages.csv]: `ship-ext` is for external components that we ship. The effect is that we don't code-sign them, but we do publish them to NuGet.org.

Comment (Eilon) [inline packages/packages.csv]: Actually, I don't think this is right. It's more nuanced - or perhaps we should split apart this package into two, like we do with libuv. The problem here is that now we don't sign *anything* in this package, but that is incorrect.

This package contains two sets of assets:
* Microsoft.Data.Sqlite.dll, which we wrote, and should be signed
* sqlite.dll, which we did not write, and should not be signed

This list doesn't accurately reflect that split.

I recommend splitting this into two packages, where we sign the contents of one, and not the other.

Comment (natemcmaster) [inline packages/packages.csv]: Splitting is also a nuanced question. See https://github.com/aspnet/Microsoft.Data.Sqlite/issues/237. Regarding signing, see https://github.com/aspnet/Coherence-Signed/pull/285

**Pull request closed without merging**